django/db/models/aggregates.py

```diff
@@ -105,6 +105,12 @@ def __init__(self, expression, distinct=False, **extra):
         super(Count, self).__init__(
             expression, distinct='DISTINCT ' if distinct else '', output_field=IntegerField(), **extra)
 
+    def __repr__(self):
+        args = self.arg_joiner.join(str(arg) for arg in self.source_expressions)
+        if self.extra['distinct'] == '':
+            return "{}({}, distinct=False)".format(self.__class__.__name__, args)
+        return "{}({}, distinct=True)".format(self.__class__.__name__, args)
```

Comment:
Is there a reason you didn't opt for something like this:
```
return "{}({}, distinct={})".format(
    self.__class__.__name__,
    self.arg_joiner.join(str(arg) for arg in self.source_expressions),
    'False' if self.extra['distinct'] == '' else 'True',
)
```
(to repeat less code)